### PR TITLE
Document Ingress Attributes

### PR DESCRIPTION
**Pull request by chrisarcand**
Ingress attributes are resources which include commit information associated with configuration versions sourced from VCS.

image

### CODE REVIEW

**Comment by chrisarcand** [comment]
Mmm. As I dwell on this more and am adding this to the TFC/E Go client, I think it may make more sense documented literally under Configuration Versions (as that is their purpose, really).

**Comment by radditude** [comment]
+1, I think that makes sense since they're not a stand-alone resource!

**Comment by chrisarcand** [comment]
@radditude Done!

**Review by nfagerlund — APPROVED** [review]
This approach seems good! 

I forget, do we associate an `ingress-attributes` with a `policy-set-versions` created from VCS? The docs for policy sets don't show that, but they also seem to be focused on handling manually created versions, rather than examining automatic ones.

**Comment by radditude** [comment]
Ah, shoot, we totally do - good catch @nfagerlund! I guess ingress attributes will need their own page after all 😞

File: content/source/docs/cloud/api/configuration-versions.html.md [hunk 1]
```diff
@@ -143,6 +143,82 @@ curl \
 }
 ```
 
+## Show a Configuration Version's commit information
```
Comment (radditude) [inline content/source/docs/cloud/api/configuration-versions.html.md:146]: Seems like we usually stick with title case for these headings!

```suggestion
## Show a Configuration Version's Commit Information
```

**Review by radditude — APPROVED** [review]

**Comment by nfagerlund** [comment]
Ah, cool, that would fit with what that comment said about it being a vestige of old TFE — we probably thought better of this when it came time to do that policy sets API.

**Comment by chrisarcand** [comment]
> Ah, cool, that would fit with what that comment said about it being a vestige of old TFE — we probably thought better of this when it came time to do that policy sets API.

Very much this ^_^ Yes, internally the modeling uses this but from the API's perspective it's just an object attribute, not a separate resource.

Thanks you two!

**Pull request merged**